src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -30,13 +30,16 @@ var ReactMultiChild = require('ReactMultiChild');
 var ReactServerRenderingTransaction = require('ReactServerRenderingTransaction');
 var {DOCUMENT_FRAGMENT_NODE} = require('HTMLNodeType');
 
+var didWarnShadyDOM = false;
```

Comment:
Ah, damn.. sorry about that. I just alpha sorted the block without realizing it wasn't a require. I'll put it back and add a newline.